docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,555 @@
+       logger.Fatal(err)
+   }
+   defer f.Close()
+
+
+   streamer, format, err := mp3.Decode(f)
+   if err != nil {
+       logger.Fatal(err)
+   }
+   defer streamer.Close()
+
+
+   speaker.Init(format.SampleRate, format.SampleRate.N(time.Second/10))
+}
+
+// Play a song
+func play(label string, logger golog.Logger) {
+   f, err := os.Open(label + ".mp3")
+   if err != nil {
+       logger.Fatal(err)
+   }
+   defer f.Close()
+
+
+   streamer, _, err := mp3.Decode(f)
+   if err != nil {
+       logger.Fatal(err)
+   }
+   defer streamer.Close()
+
+
+   done := make(chan bool)
+   speaker.Play(beep.Seq(streamer, beep.Callback(func() {
+       done <- true
+   })))
+
+
+   <-done
+}
+
+// Code Sample Connect() Code 
+func main() {
+ logger := golog.NewDevelopmentLogger("client")
+ robot, err := client.New(
+     context.Background(),
+     ".viam.cloud", // Insert your remote address here. Go to the Code Sample tab in the Viam app to find.
+     logger,
+     client.WithDialOptions(rpc.WithCredentials(rpc.Credentials{
+         Type:    utils.CredentialsTypeRobotLocationSecret,
+         Payload: "", // Insert your robot location secret here. Go to the Code Sample tab in the Viam app to find.
+     })),
+ )
+ if err != nil {
+     logger.Fatal(err)
+ }
+
+
+ defer robot.Close(context.Background())
+  
+// Get the shape classifier from the robot 
+ visService, err := vision.FromRobot(robot, "shape-classifier")
+ if err != nil {
+   logger.Error(err)
+ }
+
+ // Initialize the speaker
+ initSpeaker(logger)
+
+
+ // Classifications logic
+ for {
+   for i := 0; i < 3; i++ {
+       visService.ClassificationsFromCamera(context.Background(), "cam",  1, nil)
+   }
+
+
+   classifications, err := visService.ClassificationsFromCamera(context.Background(), "cam", 1, nil)
+   if err != nil {
+       logger.Fatalf("Could not get classifications: %v", err)
+   }
+   if len(classifications) > 0 && classifications[0].Score() > 0.7 {
+       logger.Info(classifications[0])
+       play(classifications[0].Label(), logger)
+   }
+
+
+ }
+}
+```
+
+Save your <file>play-songs.go</file> program with this logic added in.
+Run the code on your personal computer as follows:
+
+``` shell
+go run ~/<my-bedtime-songs-bot-directory>/play-songs.go
+```
+
+The full example source code for <file>play-songs.go</file> is available on [GitHub](https://github.com/viam-labs/bedtime-songs-bot/blob/main/play_songs.go).
+
+Now, as shown below, your smart bedtime songs bot knows to play a song whenever it sees a shape on the camera:
+
+{{<gif webm_src="/tutorials/img/bedtime-songs-bot/robot_babysitter.webm" mp4_src="/tutorials/img/bedtime-songs-bot/robot_babysitter.mp4" max-width="500px" alt="A demonstration of the bedtime songs bot is taking place in an office. Tess holds up brightly colored puzzle pieces in front of the camera of a Macbook laptop. As the webcam on the laptop recognizes the puzzle pieces, different songs start to play on the speakers of the computer.">}}
```

Comment:
we have this as a video with sound right? If you change this to use the video shortcode you'll actually have sound too!

Reply:
Epic, changed